Formula/go.rb

```diff
@@ -3,16 +3,20 @@ class Go < Formula
   homepage "https://golang.org"
 
   stable do
-    url "https://storage.googleapis.com/golang/go1.7.5.src.tar.gz"
-    mirror "https://fossies.org/linux/misc/go1.7.5.src.tar.gz"
-    version "1.7.5"
-    sha256 "4e834513a2079f8cbbd357502cccaac9507fd00a1efe672375798858ff291815"
+    # Version bumps of this formula require formulae that depend on Go at
+    # runtime to be rebuilt in order to pass their tests. This can be done by
+    # bumping those formula's revision.
+
+    url "https://storage.googleapis.com/golang/go1.8.src.tar.gz"
+    mirror "https://fossies.org/linux/misc/go1.8.src.tar.gz"
+    version "1.8"
+    sha256 "406865f587b44be7092f206d73fc1de252600b79b3cacc587b74b5ef5c623596"
 
     go_version = version.to_s.split(".")[0..1].join(".")
     resource "gotools" do
       url "https://go.googlesource.com/tools.git",
           :branch => "release-branch.go#{go_version}",
-          :revision => "6220cba6419b2bf78aad19d85c347ecc0fda2b53"
+          :revision => "12a9eeeaa0fb4dc5c00e510411b39ab6e4b8796b"
```

Comment:
This appears to be the hash of the tree not the top commit, and I think specifying branch at all is actually incorrect, since that's really only for head specs that use a branch other than master.